hivemq-edge/src/frontend/src/extensions/datahub/components/nodes/SchemaNode.tsx

```diff
@@ -1,28 +1,29 @@
 import { HStack, Text, VStack } from '@chakra-ui/react'
 import { FC } from 'react'
-import { Handle, NodeProps, Position } from 'reactflow'
+import { NodeProps, Position } from 'reactflow'
 
 import { DataHubNodeType, SchemaData } from '../../types.ts'
-import { styleSourceHandle } from '../../utils/node.utils.ts'
 import NodeIcon from '../helpers/NodeIcon.tsx'
 import { NodeWrapper } from './NodeWrapper.tsx'
+import { CustomHandle } from './CustomHandle.tsx'
+import { useTranslation } from 'react-i18next'
 
 export const SchemaNode: FC<NodeProps<SchemaData>> = (props) => {
-  const { id, data } = props
+  const { t } = useTranslation('datahub')
+  const { id, data, type } = props
 
   return (
     <>
       <NodeWrapper route={`node/${DataHubNodeType.SCHEMA}/${id}`} {...props}>
         <HStack>
           <NodeIcon type={DataHubNodeType.SCHEMA} />
-          <Text>Schema</Text>
+          <Text data-testid={'node-title'}> {t('workspace.nodes.type', { context: type })}</Text>
```

Comment:
suggestion: maybe we can align eslint to do that automatically

```suggestion
          <Text data-testid="node-title"> {t('workspace.nodes.type', { context: type })}</Text>
```